lib/livebook/teams.ex

```diff
@@ -223,6 +263,13 @@ defmodule Livebook.Teams do
     add_errors(change(secret), Secret.__schema__(:fields), errors_map)
   end
 
+  defp add_file_system_errors(%struct{} = file_system, errors_map) do
+    %{error_field: field} = FileSystem.external_metadata(file_system)
+    errors_map = for {_key, values} <- errors_map, into: %{}, do: {field, values}
```

Comment:
If you want something a bit more concise (it is all the same):

```suggestion
    errors_map = Map.new(errors_map, fn {_key, values} -> {field, values} end)
```